File: OIS/ois_f1/f1.c
// NOTE: it is recommended to use this even if you don't understand the following code.

#include <assert.h>
#include <stdio.h>
#include <stdlib.h>

// constraints
#define MAXN 100

// input data
int N, i;
int H[MAXN], V[MAXN];

int main() {
    //  uncomment the following lines if you want to read/write from files
    //  freopen("input.txt", "r", stdin);
    //  freopen("output.txt", "w", stdout);

    assert(1 == scanf("%d", &N));
    for (i = 0; i < N; i++)
        assert(1 == scanf("%d", &H[i]));
    for (i = 0; i < N; i++)
        assert(1 == scanf("%d", &V[i]));

    // insert your code here

    printf("%d\n", 42); // print the result
    return 0;
}

File: OIS/ois_f1/f1.java
// WARNING: this is a generic template, not specific to any task

import java.util.*;
import java.io.*;
import java.lang.*;

public class f1 {
    public static void main(String[] args) throws FileNotFoundException, IOException {
        InputStream fin = System.in;
        OutputStream fout = System.out;
        // uncomment the following lines if you want to read/write from files
        //fin = new FileInputStream("input.txt");
        //fout = new FileOutputStream("output.txt");

        Scanner scn = new Scanner(fin);
        PrintStream prnt = new PrintStream(fout);

        int N = scn.nextInt();
        int[] H = new int[N];
        for (int i = 0; i < N; i++)
            H[i] = scn.nextInt();
        int[] V = new int[N];
        for (int i = 0; i < N; i++)
            V[i] = scn.nextInt();
        // insert your code here

        prnt.format("%d\n", 42); // print the result
        fout.flush();
    }
}
